Makefile

```diff
@@ -106,28 +106,11 @@ proto: protolint protobuild
 
 .PHONY: protobuild
 protobuild:
-	# To add a new directory containing a proto pass the  proto's root path in
-	# through the --proto_path flag.
-	@bash scripts/protoc_gen_plugin.bash \
-		"--proto_path=internal/proto/local" \
-		"--proto_include_path=internal/proto/third_party" \
-		"--plugin_name=go" \
-		"--plugin_out=${TMP_DIR}"
-	@bash scripts/protoc_gen_plugin.bash \
-		"--proto_path=internal/proto/local" \
-		"--proto_include_path=internal/proto/third_party" \
-		"--plugin_name=go-grpc" \
-		"--plugin_out=${TMP_DIR}"
-	@bash scripts/protoc_gen_plugin.bash \
-		"--proto_path=internal/proto/local/" \
-		"--proto_include_path=internal/proto/third_party" \
-		"--plugin_name=grpc-gateway" \
-		"--plugin_out=logtostderr=true:${TMP_DIR}"
+	@buf generate -o "${TMP_DIR}"
```

Comment:
Is this tmp dir needed? I think buf can output to the correct location eliminating the need for the tmp dir and the copy step below

Reply:
Yea, the `source_relative` option is what I was thinking of, which actually could be done without buf, but I havnt tried it on our files to see how it changes the structure.